setup.cfg

```diff
@@ -92,8 +92,9 @@ console_scripts =
 
 [options.extras_require]
 tracing =
-    opentelemetry-exporter-jaeger
+    opentelemetry-exporter-jaeger-thrift
```

Comment:
I don't think we need to explicitly specify this, since `opentelemetry-exported-jaeger` also includes `opentelemetry-exporter-jaeger-thrift` as a dependency.

Reply:
It was just in order to minimize the number of required packages since we only import `opentelemetry.exporter.jaeger.thrift` and because there was a problem raised [on Slack](https://bentoml.slack.com/archives/CKRANBHPH/p1660564534763419?thread_ts=1660563581.202739&cid=CKRANBHPH) were thrift was not properly installed.
Could you check if it works properly for you without specifying thrift?

Reply:
I tested this out yesterday and it works on my end.

Reply:
In that case I'll remove it, thanks for testing! 👍